bin/app

```diff
@@ -8,9 +8,11 @@ require 'controler'
 require 'creator'
 require 'null_action'
 require 'user_interface'
+require 'validator'
 
 array_database = ArrayDatabase.new
-user_interface = UserInterface.new($stdin, $stdout)
+validator = Validator.new
```

Comment:
No need to create a variable for this, you can instantiate it directly in the argument of the UI